src/riak_kv_w1c_worker.erl

```diff
@@ -142,6 +142,7 @@ put(RObj, Options) ->
             Error
     end.
 
+
```

Comment:
These are the traces of an earlier effort to add a TS `delete` here, which proved to be a false lead.